src/app/pages/apptrust/apptrust.page.ts

```diff
@@ -15,18 +17,52 @@ declare var navigator: any;
 
 export class AppTrustPage implements OnInit {
   private static readonly METRICS_KEY = 'security';
+  private platform: Platform;
```

Comment:
unused variable.

Reply:
:+1:

Reply:
Actually this should cause build to fail (style) if not I will review ts-lint config.

Reply:
tslint gives a warning, but not an error:

```
Member 'serverUrl' implicitly has an 'any' type, but a better type may be inferred from usage.ts(7045)
```